demos/serverless/deploy.js

```diff
@@ -147,7 +147,10 @@ function ensureApp(appName) {
 function ensureTools() {
   spawnOrFail('aws', ['--version']);
   spawnOrFail('sam', ['--version']);
-  spawnOrFail('npm', ['install']);
+
+  // Work around https://github.com/awslabs/aws-embedded-metrics-node/issues/84.
+  // When that issue is fixed, you can remove --legacy-peer-deps.
+  spawnOrFail('npm', ['install', '--legacy-peer-deps']);
```

Comment:
Should we use `--no-save` here and down below as well?

From NPM blog on NPM 7 update:
"Running npm install with npm 7 in a project with a v1 lockfile will replace that lockfile with the new v2 format. To avoid this, you can run npm install --no-save."

So for people running on npm 7 using this dont generate the v2 lock file.

Reply:
Possibly. I'd rather we just… fix it. This change was just necessary for me to test a deployed version, because the machine I do my deploys on is running npm 7.

Reply:
Fixing as in upgrading to NPM 7 😅  ?

Reply:
Not necessarily — just upgrading to lockfile version 2. npm 6 and npm 7 both support that version.

Reply:
Cool, sounds good as it is back-compatible as well.